### DISCUSSION TITLE
Mod

### DISCUSSION THREAD

**Discussion by itzjakm** [ScratchAddons/ScratchAddons#1283] [Help]
Is there a way to apply the extension to a mod? (outside the scratch page)

**Comment by mxmou** [comment]
Some editor addons would work with mods if the appropriate URLs would be added to the extension manifest and to the addon's `addon.json`.

**Edit:** It isn't actually that simple because the code for matching URLs currently assumes they're on Scratch.

**Comment by WorldLanguages** [comment]
TurboWarp developer GarboMuffin plans to add all editor addons to TurboWarp built in in the future - there's no real reason a mod would need an extension to have addons, since it's already a mod.
